src/main/webapp/app/shared/achievement/item/achievement-item.component.html

```diff
@@ -1,12 +1,22 @@
 <ng-template #popoverContent>
     <div class="d-flex flex-column align-items-center">
         <div>{{item.name}}</div>
+        <div *ngIf="item.instantMultiplier && !active">
+            {{('teamdojoApp.level.instantMultiplier' | translate) + ': ' + item.instantMultiplier}}
+        </div>
         <small *ngIf="hasStatus">( {{progress | number: '1.0-2' }}% )</small>
     </div>
 </ng-template>
-<div class="achievement-item" (click)="selectItem($event)" [ngClass]="[active ? 'active' : '', type, itemStatusCssClass]"
+<div class="achievement-item" (click)="selectItem($event)"
+     [ngClass]="[active ? 'active' : '', type, itemStatusCssClass]"
      [ngbPopover]="popoverContent" triggers="mouseenter:mouseleave" placement="top">
+
+    <span *ngIf="item.instantMultiplier && active" class="badge badge-pill multiplier-badge">
+        {{('teamdojoApp.level.instantMultiplier' | translate) + ': ' + item.instantMultiplier}}
```

Comment:
see comment above https://github.com/iteratec/slab-dojo/pull/8/files#diff-6c9080e90c850722649ea979f90c7b92R5
```suggestion
        {{('teamdojoApp.teams.achievements.item.multiplier' | translate) + ': ' + item.instantMultiplier}}
```